setup.py

```diff
@@ -3,7 +3,7 @@
 setup(
     name='silasdk',
 
-    version='1.0.4',
+    version='2.0.0',
```

Comment:
semver convention would say this is a minor bump, so 1.0.4 -> 1.1.0, not all the way to 2.0.  This is not an incompatible change, biggest change is only dropping support for an end-of-life python version.